requirements.txt

```diff
@@ -1,10 +1,7 @@
-tornado>=3.1.1
-
-pycurl
-pylibmc
+elasticsearch
```

Comment:
You don't have to rip it out of here now, but we could toss elasticsearch in a follow on PR (then figure out what we're doing after).